test/fixer/sqs_client_test.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: utf-8
+
+require 'minitest_helper'
+
+describe Fixer::SqsClient do
+  let(:client) { Fixer::SqsClient.new }
+
+  let(:job_params) do
+    {
+      job: {
+        job_type: 'test',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.